cmd/go-tpc/tpch.go

```diff
@@ -50,6 +58,18 @@ func executeTpch(action string) {
 		runtime.GOMAXPROCS(maxProcs)
 	}
 
+	if action == "run" && tpchConfig.EnableQueryTuning {
+		serverVer, err := getServerVersion(globalDB)
+		if err != nil {
+			panic(fmt.Errorf("get server version failed: %v", err))
+		}
+		if semVer, ok := util.NewTiDBSemVersion(serverVer); ok {
+			if err := setQueryTuningVars(globalDB, semVer); err != nil {
+				panic(fmt.Errorf("set session variables failed: %v", err))
+			}
+		}
```

Comment:
Shall we add a corresponding `else` branch that prints an info log of "Not a TiDB sem version: %s"?

Reply:
I think it's not a good idea, this approach may cause confusion for non-TiDB users. 🤔

Reply:
Well, my concern is that we need some way to confirm why the tuning is enabled or not. So at least we should print something for the sake of diagnostic. Maybe simply printing the version string (NOT in `else` then)?